addons/account/static/src/components/bills_upload/bills_upload.js

```diff
@@ -45,6 +45,14 @@ AccountFileUploader.components = {
     FileUploader,
 };
 AccountFileUploader.template = "account.AccountFileUploader";
+AccountFileUploader.extractProps = ({ attrs }) => ({
+    template: attrs.template || "account.JournalUploadLink",
+    extraContext: attrs.options.extraContext || {},
+    btnClass: attrs.btnClass || "",
+    linkText: attrs.linkText || "Upload",
+});
```

Comment:
```suggestion
    const { template, options, btnClass,btnClass } = attrs
    template:template || "account.JournalUploadLink",
    extraContext:options.extraContext || {},
    btnClass:btnClass || "",
    linkText:linkText || "Upload",
});
```

Reply:
I would have to change it to a function with a `return` statement...
is that preferred?